lib/ex_doc/language.ex

```diff
@@ -135,9 +135,9 @@ defmodule ExDoc.Language do
   def get(:erlang, _module), do: {:ok, ExDoc.Language.Erlang}
 
   def get(language, module) do
-    IO.puts(
-      :stderr,
-      "warning: skipping module #{inspect(module)}, reason: unsupported language (#{inspect(language)})"
+    IO.warn(
+      "skipping module #{inspect(module)}, reason: unsupported language (#{inspect(language)})",
```

Comment:
```suggestion
      "skipping module #{module}, reason: unsupported language (#{language})",
```

Reply:
I think showing the module as is, as well as the language, will be less confusing. Other than that, this is ready to go!